### PR TITLE
test: adds the concurrent test to mimic the upload behavior

### PR DESCRIPTION
**Pull request by helio-frota**
run with `cargo test advisory::endpoints::test::upload_default_csaf_format_multiple_concurrent -- --nocapture`

Related to #999

### CODE REVIEW

File: modules/ingestor/src/graph/organization.rs [hunk 1]
```diff
@@ -83,8 +83,14 @@ impl Graph {
                 let mut entity = organization::ActiveModel::from(found.organization);
                 entity.website = Set(information.website);
                 entity.cpe_key = Set(information.cpe_key);
-                let model = entity.update(&self.connection(&tx)).await?;
-                Ok(OrganizationContext::new(found.graph, model))
+                let model = match entity.update(&self.connection(&tx)).await {
+                    Ok(model) => Ok(model),
+                    Err(e) => {
+                        eprintln!("Failed to update organization: {:?}", e);
```
Comment (ctron) [inline modules/ingestor/src/graph/organization.rs:89]: We have some guidelines on how to deal with this: https://github.com/trustification/trustify/blob/main/docs/design/log_tracing.md

**Review by ctron — CHANGES_REQUESTED** [review]

File: modules/ingestor/src/graph/organization.rs (hunk 1 above)
Comment (helio-frota) [reply modules/ingestor/src/graph/organization.rs:89]: :facepalm: thanks, and tbh I forgot to remove `eprintln!`

I'll update based on this statement:
`Log messages and tracing should be relevant to operating personell and developers, not end users.` with log::error! macro thanks :+1:

File: modules/ingestor/src/graph/organization.rs [hunk 2]
```diff
@@ -83,8 +83,14 @@ impl Graph {
                 let mut entity = organization::ActiveModel::from(found.organization);
                 entity.website = Set(information.website);
                 entity.cpe_key = Set(information.cpe_key);
-                let model = entity.update(&self.connection(&tx)).await?;
-                Ok(OrganizationContext::new(found.graph, model))
+                let model = match entity.update(&self.connection(&tx)).await {
+                    Ok(model) => Ok(model),
+                    Err(e) => {
+                        log::error!("Failed to update organization: {e}");
```
Comment (ctron) [inline modules/ingestor/src/graph/organization.rs:89]: Unless this terminates or seriously affects the the application, this should be reduced to info or debug.

**Comment by helio-frota** [comment]
I'll revert the `modules/ingestor/src/graph/organization.rs` code and since the test is already ignored we can safe-merge-it

**Pull request closed without merging**